src/dsl/interaction.spec.ts

```diff
@@ -1,12 +1,16 @@
 import * as chai from "chai";
-const expect = require("chai").expect;
-import { Interaction } from "./interaction";
+import * as chaiAsPromised from "chai-as-promised";
+import {HTTPMethod} from "../common/request";
+import {Interaction, RequestOptions} from "./interaction";
```

Comment:
I've just pushed up an updated linting config that should ensure consistency of the padding around braces (amongst others). I'm guessing your editor doesn't like the padding.